[2023-03-02T21:00:33.598] [ERROR] default - Error: Invalid login: 535 Authentication failed
    at SMTPConnection._formatError (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:1548:34)
    at SMTPConnection.<anonymous> (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:953:20)
    at SMTPConnection._onData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10) {
  code: 'EAUTH',
  response: '535 Authentication failed',
  responseCode: 535,
  command: 'AUTH PLAIN'
}
[2023-03-02T21:00:34.563] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-02T21:00:35.312] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-04T03:07:40.071] [ERROR] default - Error: Invalid login: 535 Authentication failed
    at SMTPConnection._formatError (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:1548:34)
    at SMTPConnection.<anonymous> (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:953:20)
    at SMTPConnection._onData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10) {
  code: 'EAUTH',
  response: '535 Authentication failed',
  responseCode: 535,
  command: 'AUTH PLAIN'
}
[2023-03-04T03:07:41.098] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-04T03:07:41.828] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-04T04:14:17.846] [ERROR] default - Error: Invalid login: 535 Authentication failed
    at SMTPConnection._formatError (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:1548:34)
    at SMTPConnection.<anonymous> (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:953:20)
    at SMTPConnection._onData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10) {
  code: 'EAUTH',
  response: '535 Authentication failed',
  responseCode: 535,
  command: 'AUTH PLAIN'
}
[2023-03-04T14:47:43.870] [ERROR] default - Error: Invalid login: 535 Authentication failed
    at SMTPConnection._formatError (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:1548:34)
    at SMTPConnection.<anonymous> (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:953:20)
    at SMTPConnection._onData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10) {
  code: 'EAUTH',
  response: '535 Authentication failed',
  responseCode: 535,
  command: 'AUTH PLAIN'
}
[2023-03-04T20:26:04.411] [ERROR] default - Error: Invalid login: 535 Authentication failed
    at SMTPConnection._formatError (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:1548:34)
    at SMTPConnection.<anonymous> (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:953:20)
    at SMTPConnection._onData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10) {
  code: 'EAUTH',
  response: '535 Authentication failed',
  responseCode: 535,
  command: 'AUTH PLAIN'
}
[2023-03-04T20:26:05.373] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-04T20:26:06.078] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-04T20:31:39.573] [ERROR] default - Error: Invalid login: 535 Authentication failed
    at SMTPConnection._formatError (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:1548:34)
    at SMTPConnection.<anonymous> (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:953:20)
    at SMTPConnection._onData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10) {
  code: 'EAUTH',
  response: '535 Authentication failed',
  responseCode: 535,
  command: 'AUTH PLAIN'
}
[2023-03-04T20:31:40.486] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-04T20:31:41.234] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-04T21:02:43.890] [ERROR] default - Error: Invalid login: 535 Authentication failed
    at SMTPConnection._formatError (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:1548:34)
    at SMTPConnection.<anonymous> (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:953:20)
    at SMTPConnection._onData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10) {
  code: 'EAUTH',
  response: '535 Authentication failed',
  responseCode: 535,
  command: 'AUTH PLAIN'
}
[2023-03-04T21:02:44.955] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-04T21:02:45.674] [ERROR] default - RestException [Error]: Authenticate
    at success (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2023-03-05T12:38:46.457] [ERROR] default - ReferenceError: usuarioId is not defined
    at completarOrdenDTO.completeData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/dtos/ordenes/completarOrden.dto.js:19:48)
    at getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:27:42)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:284:15
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:421:3)
[2023-03-05T12:39:41.453] [ERROR] default - ReferenceError: usuarioId is not defined
    at completarOrdenDTO.completeData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/dtos/ordenes/completarOrden.dto.js:21:48)
    at getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:27:42)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:284:15
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:421:3)
[2023-03-05T12:39:42.858] [ERROR] default - ReferenceError: usuarioId is not defined
    at completarOrdenDTO.completeData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/dtos/ordenes/completarOrden.dto.js:21:48)
    at getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:27:42)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:284:15
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:421:3)
[2023-03-05T12:39:55.593] [ERROR] default - ReferenceError: usuarioId is not defined
    at completarOrdenDTO.completeData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/dtos/ordenes/completarOrden.dto.js:21:48)
    at getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:27:42)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:284:15
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:421:3)
[2023-03-05T12:47:48.112] [ERROR] default - ReferenceError: completarOrdenDTO is not defined
    at getCompleteOrder (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/services/ordenes.service.js:22:22)
    at getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:29:33)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:284:15
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:421:3)
[2023-03-05T12:47:59.881] [ERROR] default - ReferenceError: completarOrdenDTO is not defined
    at getCompleteOrder (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/services/ordenes.service.js:22:22)
    at getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:29:33)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:284:15
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (C:\Users\leand\Desktop\Proyectos_program\BackEnd_32105-LeandroTalenton\node_modules\express\lib\router\index.js:421:3)
[2023-03-05T12:48:14.267] [ERROR] default - ReferenceError: completarOrdenDTO is not defined
    at getCompleteOrder (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/services/ordenes.service.js:22:22)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:29:27)
[2023-03-05T12:49:47.870] [ERROR] default - ReferenceError: completarOrdenDTO is not defined
    at getCompleteOrder (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/services/ordenes.service.js:23:22)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:29:27)
[2023-03-05T12:50:09.429] [ERROR] default - TypeError: Found non-callable @@iterator
    at getCompleteOrder (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/services/ordenes.service.js:24:22)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:29:27)
[2023-03-05T12:51:27.582] [ERROR] default - ReferenceError: usuarioId is not defined
    at completarOrdenDTO.completeData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/dtos/ordenes/completarOrden.dto.js:21:48)
    at getCompleteOrder (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/services/ordenes.service.js:25:42)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:29:27)
[2023-03-05T12:52:15.770] [ERROR] default - ReferenceError: res is not defined
    at getCompleteOrder (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/services/ordenes.service.js:27:5)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async getOrderData (file:///C:/Users/leand/Desktop/Proyectos_program/BackEnd_32105-LeandroTalenton/src/controllers/ordenes.controller.js:29:27)
